### PR TITLE
4652 remove immunisation program oms mutations

### PR DESCRIPTION
**Pull request by lache-melvin**
Start of #4652

# 👩🏻‍💻 What does this PR do?

THIS IS THE AGGRESSIVE DELETION PR

We are moving to managing immunisation programs through mSupply, like other programs (needed to loop in with patient program functionality)

So no need to create/delete programs through OMS

## 💌 Any notes for the reviewer?

Kept the update mutation because I figured we'll probably use something close to that for the adding age ranges etc. - probably faster than having to reimplement from scratch

Not much to review, just check she all still works. Just wanted all this noise separate from actual code changes 😁 

# 🧪 Testing

- [ ] Immunisation programs view on OMS central has no buttons to do anything with 😁 

# 📃 Documentation

- [x] **Part of an epic**: documentation will be completed for the feature as a whole
- [ ] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [ ] **These areas should be updated or checked**: 
  1.
  2.

### CODE REVIEW

File: server/service/src/stock_line/tests/query.rs [hunk 1]
```diff
@@ -22,14 +22,14 @@ mod query {
             service.get_stock_lines(
                 &context,
                 Some(PaginationOption {
-                    limit: Some(2000),
+                    limit: Some(6000),
```
Comment (lache-melvin) [inline server/service/src/stock_line/tests/query.rs:25]: This test was failing in develop, looks like max limit got increased?

**Comment by Chris-Petty** [comment]
Looks nuked to me!

Tests pass locally. Some fluff the compiler is complaining about you might want to trim out ?

image

**Review by Chris-Petty — APPROVED** [review]

**Comment by lache-melvin** [comment]
did some more deleting - could you double check happy with deleting the deleted datetime 😅

**Review by Chris-Petty — APPROVED** [review]
Hmm LGTM. Was deleted date time just programs that you make independently of the OG central server and master list (and `master_list.is_active`)?

**Comment by lache-melvin** [comment]
> Hmm LGTM. Was deleted date time just programs that you make independently of the OG central server and master list (and `master_list.is_active`)?

Yep was only added for the independent immunisation programs 👍 Had a quick look and don't think we check whether master list is active in current OMS program queries... might need to at some point but I think a separate issue 🙏

**Pull request merged**